src/components/layout/Header.js

```diff
@@ -194,6 +197,10 @@ const InfoBox = styled.div`
   border: ${({ type }) =>
     `${selectBorderStyle(type)} 5px ${selectTypeColor(type)}`};
 `
+const CtaBox = styled(Box)`
```

Comment:
@ric9176 I'd use instead of `${space({ p: 4 })}` directly in the style the following:
```
CtaBox.defaultProps = {
    p: 4
}
```
This way you don't have to import space from styled-components, and that's an implementation detail in the Box